bin/projects/dbatools/dbatools/dbatools.csproj

```diff
@@ -40,15 +40,15 @@
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="5.1.0-preview1.22279.3" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="5.0.1" />
```

Comment:
There are probably times you want to use the preview release of the ADO.NET dll here, but generally, you don't need those breaking changes (says the guy who was giving always encryptes talks for years before it was supported in .net core)

Reply:
seems reasonable